frontend/src/app/Checkin/Start.tsx

```diff
@@ -0,0 +1,79 @@
+import { StatusBar, StyleSheet, Text, View } from "react-native";
+
+import Mascots from "@/assets/Poc_Mascots.svg";
+import Button from "@/components/Button";
+
+import { lightModeColors } from "@/constants/colors";
+
+export default function Start() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.topSection}>
+        <Mascots style={styles.logo} />
+        <Text style={styles.title}>Let's check in on you</Text>
+        <Text style={styles.text}>
+          Lorem ipsum odor amet, consectetuer adipiscing elit. Curae phasellus laoreet nullam in
```

Comment:
Also can we replace this placeholder content lol, just put smth like "We'll ask a short series of questions about your past week"